pkg/component/application/jira/v0/component_test.go

```diff
@@ -35,17 +35,29 @@ func TestComponent_ListBoardsTask(t *testing.T) {
 	testcases := []TaskCase[ListBoardsInput, ListBoardsOutput]{
 		{
 			_type: "ok",
-			name:  "get all boards",
+			name:  "list all boards",
 			input: ListBoardsInput{
 				MaxResults: 10,
 				StartAt:    0,
 			},
 			wantResp: ListBoardsOutput{
-				Total:      1,
+				Total:      3,
 				StartAt:    0,
 				MaxResults: 10,
 				IsLast:     true,
 				Boards: []Board{
+					{
```

Comment:
@Ping-Lin Chang this means that your implementation of `FillInDefaultValues` isn't filling the default value for the `board-type` parameter (`"simple"`). Indeed there are three boards but the result should return only one because the [task by default filters the simple boards](https://github.com/instill-ai/pipeline-backend/blob/main/pkg/component/application/jira/v0/config/tasks.json#L227).

Reply:
Good point. I will refactor `FillInDefaultValues` completely and ensure this test can pass originally.